### ISSUE TITLE
Exports not defined after 4.3.0/4.3.1

### ISSUE THREAD

**Issue by mscienski** [mobxjs/mobx-react#329]
Importing from mobx-react fails after 4.3.0

```
Uncaught ReferenceError: exports is not defined
  at http://localhost:9878webpack:///node_modules/mobx-react/index.module.js:857:171
```

The line in question:
`export { propTypes, propTypes as PropTypes, onError, observer, Observer, renderReporter, componentByNodeRegistery, trackComponents, useStaticRendering, Provider, inject };export default exports;`
The line is the only instance of `exports` in the file. I assume something in the rollup config changed the default export, and it just happens to be undefined now.

**Comment by MBach** [comment]
Same here, this is a blocker issue for me!

**Comment by aa1011372655** [comment]
Well, I have this problem, can only write dead version of the downgrade, and hope to quickly repair

**Comment by mweststrate** [comment]
@mscienski  / @MBach  could you verify whether `mobx-react@4.3.2-rc.1` fixes this?

**Comment by mscienski** [comment]
@mweststrate `4.3.2-rc.1` works for me. 👍

**Comment by mweststrate** [comment]
Released 4.3.2

**Comment by MBach** [comment]
Good job 👍
